docs/user-guides/client_config_guide.md

```diff
@@ -1,12 +1,138 @@
 # Client Configuration Guide
 
+**Note:** For Installation and Starting of the Mumble Client, see the specific [Userguides](README.md) for your Operating System.
 
-<!-- Pictures or even Gifs could be added for better presentation-->
+<!-- Note: The pictures below are from a Mumble Client on Arch Linux, using the Dark Standard Theme, with already adjusted options. -->
+
+<!-- Todo -->
+<!--
+
+- Add Troubleshooting section or seperate guide?
+
+- Pictures or even Gifs could be added for better presentation
+ 
+![Alt-Text](/Path/to/picture.jpg)
+Here should be a picture
+
+--->
 
 
 <!--## Audio Wizard? -->
 
 <!--## Important Config Options -->
 
-<!-- Topics suggested by Krzmbrzl: 
-change theme between Lite and Dark mode, echo cancellation, transmission modes (VAD, PTT, continous), shortcuts -->
\ No newline at end of file
+<!-- 
+Topics suggested by Krzmbrzl: 
+- change theme between Lite and Dark mode, 
+- (done) echo cancellation, 
+- (almost done)transmission modes (VAD, PTT, continous),
+- shortcuts 
+
+-->
+
+## Configuration Menu
+
+<!-- where to find config options, add gif? -->
+
+Click on `Configure`->`Settings` or on the `gear`-symbol to open the Configuration Menu.
```

Comment:
```suggestion
Click on `Configure`->`Settings` or on the gear-symbol to open the Configuration Menu.
```
Highlighting this as code as well creates the impression that "gear" is also a keyword the user has to search for

Reply:
That is not the recent version of this text. I included the symbol itself, it is a bit big though, so I need to find ways to make it smaller.
Most likely html code; because markdown's image size is maybe not supported everywhere.